exampleSite/config.toml

```diff
@@ -64,7 +64,8 @@ buildFuture = true
 
 [params]
 mainSections = ["episode"]
-
+episode_number_style  = "parens"
```

Comment:
since the example in REFERENCE.md is "brackets", I would suggest making the exampleSite use the same value